### ISSUE TITLE
Search button does nothing

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#5859]
## Area or Component

Device code lookup > Search button
Conduct Test > Search button

## Quick Summary

Clicking the button does nothing

## Accessibility Violation

[Role is incorrect](https://docs.deque.com/issue-help/1.0.0/en/aria-role-missing) 
https://cdc-devtools.dequecloud.com/issues/0c87f67d-8b8a-4da2-9278-3049c9687de4

## Action Requested

Remove the search button or provide it an action
If there are other places on the site where we have a search button that does nothing, see what's happening there and let's try to have same (accessible) behavior across the whole site

## Screenshots (optional)

https://github.com/CDCgov/prime-simplereport/assets/10108172/00253b70-5ab3-414d-bee1-13bb2c6fe094

## Source Code (optional)

https://github.com/CDCgov/prime-simplereport/blob/17c28123b9674ffbe4eef6b47ebcdf960d32d87e/frontend/src/app/uploads/DeviceLookup/DeviceLookup.tsx#L124-L130
https://github.com/CDCgov/prime-simplereport/blob/17c28123b9674ffbe4eef6b47ebcdf960d32d87e/frontend/src/app/testQueue/addToQueue/AddToQueueSearch.tsx#L266-L272
